en/user_guide/node.rst

```diff
@@ -0,0 +1,41 @@
+Node
+====
+
+Languages
+---------
+
+A node has several languages defined during the configuration of Back Office.
+Clicked on the language tab to change the node language. The first time it will create your node in this language.
+
+.. image:: ../../images/languages.png
+
+Workflow
+--------
+
+Node have a status and can have a validation workflow, that is stage to be crossed before you can display it on Front Office.
```

Comment:
Nodes
"that is stage to be crossed" what do you mean ?
Maybe something like "which is the series of steps to achieve for publication of a content" ?